Nighthold/Locales/koKR.lua

```diff
@@ -71,6 +71,8 @@ end
 
 L = BigWigs:NewBossLocale("Gul'dan", "koKR")
 if L then
+	L.warmup_trigger = "부서진 해변에서의 치욕을 잊었느냐?"

```

Comment:
Is this the whole yell? If it isn't, just add the whole yell next to it commented out `--`